profiles/Sarvatt-setup.yaml

```diff
@@ -254,6 +268,7 @@ training_abilities:
 #  Collect: 200
   Analyze: 121
   Locks: 70
+  Almanac: 600
```

Comment:
❓ Should this value be lower so it checks if the timer is up more frequently? I'm envisioning a worst-case scenario in which the timer is a few seconds from being up, the training ability is checked (and skipped since the timer is not quite up yet), then you wait an additional 10 minutes before using the almanac.

Reply:
indeed, wont hurt to lower the timer since it checks the uservar